src/Builder/AbstractFormContractor.php

```diff
@@ -159,8 +159,8 @@ private function isAnyInstanceOf(?string $type, array $classes): bool
 
         // handle form type inheritance and check all parent types
         $resolvedType = $this->formRegistry->getType($type);
-        if (null !== $resolvedType->getParent()) {
-            $parentType = \get_class($resolvedType->getParent()->getInnerType());
+        if (null !== $parentType = $resolvedType->getParent()) {
```

Comment:
Personnally, I prefer to avoid defining variable in a if, but seems like no linter is against this.

Reply:
I don't mind changing it :blush: Shall I?